OCast/src/main/java/org/ocast/core/utils/JsonTools.kt

```diff
@@ -31,7 +31,8 @@ import com.fasterxml.jackson.module.kotlin.readValue
 
 object JsonTools {
 
-    val objectMapper = jacksonObjectMapper()
+    @PublishedApi
+    internal val objectMapper = jacksonObjectMapper()
```

Comment:
can be private : `private val objectMapper = jacksonObjectMapper()` if inline class is internal :` internal inline fun <reified T : Any> decode(json: String): T`